### PR TITLE
React Table Created with Sorting and Filtering Functionalities

### PR DESCRIPTION
**Pull request by jamesli898**
-[ ] React table was formatted for the orders. We added the filtering + sorting functionalities for the order table. Now we are waiting for all the other PRs to go through in order to style, and then also format the table for inventory.
75594417_493374431251847_6708382416886562816_n

### CODE REVIEW

**Comment by jamesli898** [comment]
Updated

File: src/App.js [hunk 1]
```diff
@@ -4,13 +4,14 @@ import Announcements from "./components/Announcements";
...
 import NavigationBar from "./components/NavigationBar";
-import { Route, Switch } from "react-router-dom";
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
 
 function App() {
   return (
     <div className="App">
       <NavigationBar />
       <Switch>
+        {/*<Route exact path="/" component={Announcements} />*/}
```
Comment (trev4ev) [inline src/App.js]: Let's uncomment this so that we can have it default to announcements

File: src/components/OrderDataTable.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+              Header: "",
+              columns: [
+                {
+                  id: "Pick Up Date",
+                  Header: "Pick Up Date",
+                  accessor: row => `${row.PickUpDate}`,
+                  filterMethod: (filter, row) =>
+                    //row._original.PickUpDate.startsWith(filter.value)
+                    {
+                      //console.log(row);
```
Comment (trev4ev) [inline src/components/OrderDataTable.js]: Can we remove this commented lines?

File: src/components/OrderDataTable.css [hunk 3]
```diff
@@ -0,0 +1,513 @@
+body {
```
Comment (trev4ev) [inline src/components/OrderDataTable.css:1]: If you end up up reusing these styles for inventory table then this file should be renamed. If there are different styles for inventory and order table then there should also be two separate css files, one for each table

File: src/components/OrderDataTable.js [hunk 4]
```diff
@@ -0,0 +1,90 @@
+import * as React from "react";
+import "./OrderDataTable.css";
+import ReactTable from "react-table";
+//import "react-table/react-table.css";
```
Comment (trev4ev) [inline src/components/OrderDataTable.js]: remove if we no longer need

File: src/components/Orders.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import "./Orders.css";
-import DataTable from "./DataTable";
+import OrderDataTable from "./OrderDataTable";
```
Comment (trev4ev) [inline src/components/Orders.js:2]: If we create a new Component for InventoryTable as well then you should probably remove DataTable

File: src/components/Orders.js [hunk 6]
```diff
@@ -26,9 +26,21 @@ class Orders extends React.Component {
...
+        Order: item[3],
+        SizeOrStyle: item[4],
+        ItemOrdered: item[5],
+        ItemQuantity: item[6]
+      };
+    });
     return (
       <div className="Orders">
-        <DataTable headings={headings} rows={this.state.orderRows} />
+        <h1> Orders </h1>
```
Comment (trev4ev) [inline src/components/Orders.js]: this should have a class name so you can apply a generic heading style

**Review by trev4ev — CHANGES_REQUESTED** [review]
Looks great! Added a couple of comments that you should look over and make changes for some before I approve

File: src/components/Post.js [hunk 7]
```diff
@@ -6,7 +6,10 @@ class Post extends React.Component {
     return (
       <div className="Post-box">
         <h1 className="Post-title">{this.props.title}</h1>
-        <p className="Post-date">{this.props.date}</p>
+        <div className="Post-name-date">
+          <p className="Post-name">{this.props.name}</p>
```
Comment (abaruah117) [inline src/components/Post.js:10]: in the future this should be a separate PR, since this is announcements

File: src/App.js (hunk 1 above)
Comment (abaruah117) [reply src/App.js]: don't uncomment, just set path of <Route component={Announcements} path="/announcements" /> to 
Route component={Announcements} path="/" 
 and move it to after
Route component={Inventory} path="/inventory" 
Route component={Orders} path="/orders"

**Review by abaruah117 — CHANGES_REQUESTED** [review]
Looks good, minor changes

**Review by trev4ev — DISMISSED** [review]

File: src/App.js [hunk 8]
```diff
@@ -4,16 +4,16 @@ import Announcements from "./components/Announcements";
 import Inventory from "./components/Inventory";
 import Orders from "./components/Orders";
 import NavigationBar from "./components/NavigationBar";
-import { Route, Switch } from "react-router-dom";
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
```
Comment (abaruah117) [inline src/App.js]: why do u import browser router or link

**Review by trev4ev — APPROVED** [review]

**Review by abaruah117 — APPROVED** [review]

**Pull request merged**